src/app.js

```diff
@@ -170,6 +231,38 @@ app.on('ready', () => {
     }
   }
 
+  function setNotificationTime(event, time) {
+    notificationWindow.close();
+    if (time && time !== store.get('notification.time')) {
+      let hours = time.split(':')[0];
+      let minutes = time.split(':')[1];
+      store.set('notification.time', time);
+      store.set('notification.hours', hours);
+      store.set('notification.minutes', minutes);
+      log.info(`Store updated - time=${time} hours=${hours} minutes=${minutes}`);
+      job.setTime(new CronTime(`0 ${minutes} ${hours} * * *`));
+    }
+  }
+
+  const job = new CronJob({
+    cronTime: `0 0 20 * * *`,
+    onTick: async function() {
+      const data = await contribution(store.get('username'));
+      if (data.currentStreak === 0 && Notification.isSupported()) {
+        new Notification({
+          title: 'Streaker',
+          body: 'You didn\'t contribute today',
+          icon: icon.icon
+        }).show()
+      }
+    }
+  });
+  
+  if (store.get('notification.isEnabled')) {
+    job.start();
+    job.setTime(new CronTime(`0 ${store.get('notification.minutes')} ${store.get('notification.hours')} * * *`));
```

Comment:
Same here, because we call `setTime()` after we start the time it actually stops it right after